### DISCUSSION TITLE
Add proper quoting to the forums.

### DISCUSSION THREAD

**Discussion by Ben-Ghazi** [RetroAchievements/RAWeb#721] [Ideas]
Responding to people with long-winded messages looks pretty sloppy right now, and at a glance could be mistaken as being part of your message.

Adding something like this:
![](https://files.vbulletin.com/3.6/plaintext-parser-example.png)

or even something as simple as Reddit's quoting: 
![text](https://user-images.githubusercontent.com/39404689/40871752-6e42d072-6607-11e8-8c69-fe9b0309adff.PNG)

would make the forums a fair bit more attractive in terms of readability.

**Comment by meleu** [comment]
~~Our~~ My intention is to make it use the markdown format.

My personal reference for a good forum software is the one used on the [RetroPie forums](https://retropie.org.uk/forum/).

**Comment by Jamiras** [comment]
This was implemented in #3074
